framework/src/modules/dpos_v2/types.ts

```diff
@@ -32,6 +32,22 @@ export interface ModuleConfig {
 	tokenIDDPoS: TokenIDDPoS;
 }
 
+export interface ModuleConfigJSON {
```

Comment:
you could use
```
export type ModuleConfigJSON = JSONObject<ModuleConfig>;
```

Reply:
Have changed the code based on the suggestions. Thanks.